book/03-git-branching/sections/rebasing.asc

```diff
@@ -201,8 +201,8 @@ So instead of the result we see in <<_merge_rebase_work>>, we would end up with
 .Rebase on top of force-pushed rebase work.
 image::images/perils-of-rebasing-5.png[Rebase on top of force-pushed rebase work.]
 
-This only works if C4 and C4' that your partner made are almost exactly the same patch.
-Otherwise the rebase won't be able to tell that it's a duplicate and will add another C4-like patch (which will probably fail to apply cleanly, since the changes would already be at least somewhat there).
+This only works if `C4` and `C4'` that your partner made are almost exactly the same patch.
+Otherwise the `rebase` won't be able to tell that it's a duplicate and will add another C4-like patch (which will probably fail to apply cleanly, since the changes would already be at least somewhat there).
```

Comment:
The word "rebase" here is just referring to the rebasing operation, so it shouldn't be monospace.

Reply:
My bad, fixed.